src/ec/packages/ec-component-social-media-share/social-media-share.story.js

```diff
@@ -9,26 +15,125 @@ import demoData from './demo/data';
 import SocialMediaShare from './ecl-social-media-share.html.twig';
 import notes from './README.md';
 
-demoData.links.forEach(link => {
-  if (link.icon) {
-    link.icon.forEach(icon => {
-      icon.path = defaultSprite; // eslint-disable-line no-param-reassign
-    });
-  }
-});
+const prepareSocialMediaShare = data => {
+  data.description = text(
+    'description',
+    data.description,
+    buttonLabels.required
+  );
+
+  data.links.forEach((link, i) => {
+    let label = buttonLabels.optional;
+    // Firt item we make it mandatory and not removable.
+    if (i === 0) {
+      label = buttonLabels.required;
+      link.label = select(
+        `links[${i}].link.label`,
```

Comment:
to me the label should be `links[${i}].label`?
same for all other labels, regarding the data in the notes we don't have `link`

Reply:
you're super right! ;) but i did the same crap in the social network follow and you merged it, this was a copy&paste :)
I'll fix it also there ;)

Reply:
I was less vigilant in the social network..